core/backend/src/BlobContainerService.ts

```diff
@@ -0,0 +1,142 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module BlobContainers
+ */
+
+// spell:ignore datacenter
+
+import { AccessToken, Id64String } from "@itwin/core-bentley";
+
+/**
+ * Types and functions for creating, deleting and authorizing access to cloud-based blob containers for an iTwin.
+ * @beta
+ */
+export namespace BlobContainer {
```

Comment:
@wgoehrig @MichaelSwigerAtBentley This interface is the outline for what the REST service for the BlobContainer service should like like.